source/reporters/ut_xunit_reporter.tpb

```diff
@@ -130,5 +129,10 @@ create or replace type body ut_xunit_reporter is
            || '" time="' || ut_utils.to_xml_number_format(a_item.execution_time()) || '" ';
   end;
 
+  overriding member function get_description return varchar2 as
+  begin
+    return 'Used for reporting test results with CI servers like Jenkins/Hudson/Teamcity.';
```

Comment:
`Provides outcomes in a format conforming with JUnit 4 and above as defined in:
 https://gist.github.com/kuzuha/232902acab1344d6b578`